resources/help/prompt.md

```diff
@@ -14,6 +14,25 @@ Returns the line currently typed into the prompt
 
 ***prompt.set(input)***
 Sets the line typed in the prompt. Replacing any current data.
+This set the cursor position to the end of the new prompt.
+
+##
+
+***prompt.get_cursor_pos() -> number***
+Gets the cursor position in the input prompt
+
+- Returns the current index of the cursor starting at 0
+
+##
+
+***prompt.set_cursor_pos(pos)***
```

Comment:
If you agree with my suggestion to use Lua indexing we'll also want to account for that w/ `set_cursor_pos(pos)` for symmetry.

Reply:
Done